tests/testthat/test-fetch_extract_and_enrichment_functions.R

```diff
@@ -528,4 +528,20 @@ if (Sys.getenv("TEST_PROTTI") == "true") {
     expect_equal(as.data.frame(table(af_domains$domain))$Freq[1], 6)
     expect_equal(as.data.frame(table(af_domains$domain))$Freq[2], 55)
   })
+
+  test_that("fetch_interpro works", {
+    unis <- c("P36578", "O43324", "Q00796", "O32583")
+    domains <- fetch_interpro(unis)
+    residues <- fetch_interpro(unis,
+      return_residue_info = TRUE
+    )
+    # Test domain info
+    expect_is(domains, "data.frame")
+    expect_equal(nrow(domains), 27)
```

Comment:
You dont need to change it. but maybe we should consider to write the tests slightly different, so that the numbers of rows are more flexibel in case the database gets updated - maybe changing to a range of expected number of rows that we would consider plausiblel

Reply:
In general I agree. It would need to be fixed for more of the fetch functions then. I think however, that it does not happen often, that the output changes. so for now I would keep it that way and if we come up with a safe way to check this in the future we can maybe change it.